client/stylesheets/_toCat.scss

```diff
@@ -2991,6 +2991,10 @@ i:hover {
     margin-top: 20px !important;
 }
 
+.managerOfferButton ul li {
+    display: inline-block !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces